region/crateria/central/Landing Site.json

```diff
@@ -1859,14 +1858,6 @@
         "The jump can be set up by using the full runway with a one-tap shortcharge, with the tap being at the top of the lowest slope."
       ]
     },
-    {
-      "id": 4,
-      "name": "Blue Space Jump (Bottom)",
-      "note": [
-        "Starting near the right runway, run through the bomb block passage, then jump right after exiting.",
-        "Using HiJump and space jump, Samus is able to elevate enough to break through the bomb blocks blocking the Gauntlet entrance."
-      ]
```

Comment:
This one is not really very useful, as you can shinespark from the plateau then refill at the ship. I think it would mostly be for players who can't really short charge, or preserving a flash suit - so it doesn't seem notable